pkg/config/policies.go

```diff
@@ -43,29 +43,13 @@ func (c *ConfigTx) UpdateConsortiumChannelCreationPolicy(consortiumName string,
 	return nil
 }
 
-// GetPoliciesForConsortiums returns a map of policies for channel consortiums.
-func (c *ConfigTx) GetPoliciesForConsortiums() (map[string]Policy, error) {
-	consortiums, ok := c.base.ChannelGroup.Groups[ConsortiumsGroupKey]
-	if !ok {
-		return nil, errors.New("consortiums missing from config")
-	}
-
-	return getPolicies(consortiums.Policies)
-}
-
-// GetPoliciesForConsortium returns a map of policies for a specific consortium.
-func (c *ConfigTx) GetPoliciesForConsortium(consortiumName string) (map[string]Policy, error) {
-	consortium, ok := c.base.ChannelGroup.Groups[ConsortiumsGroupKey].Groups[consortiumName]
+// GetPoliciesForConsortiumOrg returns a map of policies for a specific consortium org.
+func (c *ConfigTx) GetPoliciesForConsortiumOrg(consortiumName, orgName string) (map[string]Policy, error) {
+	consortium, ok := c.updated.ChannelGroup.Groups[ConsortiumsGroupKey].Groups[consortiumName]
 	if !ok {
 		return nil, fmt.Errorf("consortium %s does not exist in channel config", consortiumName)
```

Comment:
I can agree with the retrieval function points here. It logically makes more sense that you could get away with returning an empty map and no error, but I still think when attempting to add or remove, an error should be returned since the updated state of the config transaction would be contrary to what the consumer had expected.

Reply:
Here, we have different error messages like consortium does not exist or unmarshal error, etc...  If an error is necessary, error types would help. If the consumer needs to have logic like `if-then-else` based on error types, now, they have to do string matching.

Reply:
I think the idea was that the consumer wouldn't have to further process any errors received from the library. I do see your point though if say the user wanted to make a CLI with different exit codes for various failures.

Reply:
Going to merge this for now, we can revisit this in a followup when we address other places that return errors and how we should expect the user flow